overlord/devicestate/devicemgr.go

```
@@ -995,6 +995,28 @@ func (m *DeviceManager) Systems() ([]*System, error) {
 
 var ErrUnsupportedAction = errors.New("unsupported action")
 
+func (m *DeviceManager) RequestReboot(systemLabel, mode string) error {
+	// most simple case: just reboot
+	if systemLabel == "" && mode == "" {
+		m.state.Lock()
+		defer m.state.Unlock()
+
+		logger.Noticef("restarting system")
+		m.state.RequestRestart(state.RestartSystemNow)
+		return nil
+	}
+
+	// no systemLabel means "current" so get the current system label
+	if systemLabel == "" {
+		systemMode := m.SystemMode()
+		currentSys, _ := currentSystemForMode(m.state, systemMode)
```

I don't we should ignore the error here

---

This is no longer ignored in my code.